SharedPackages/DataBrokerProtectionCore/Sources/DataBrokerProtectionCoreTestsUtils/Mocks.swift

```diff
@@ -631,6 +631,10 @@ public final class DataBrokerProtectionSecureVaultMock: DataBrokerProtectionSecu
         return brokers
     }
 
+    public func fetchAllNonRemovedBrokers() throws -> [DataBroker] {
+        return brokers
```

Comment:
This should probably filter the removed ones

Reply:
Yeah, technically it’s not required since we set the content of the `brokers` property on the mock, but it’s probably more semantic this way, right? I’ll update it.